examples/nlp/text_classification/conf/text_classification_config.yaml

```diff
@@ -23,76 +23,67 @@ trainer:
   max_steps: null # precedence over max_epochs
   accumulate_grad_batches: 1 # accumulates grads every k batches
   amp_level: O0 # O1/O2 for mixed precision
+  precision: 32 # Should be set to 16 for O1 and O2
   distributed_backend: ddp
   checkpoint_callback: False  # Provided by exp_manager
   logger: False  # Provided by exp_manager
   row_log_interval: 1  # Interval of logging.
   val_check_interval: 1.0  # Set to 0.25 to check 4 times per epoch, or an int for number of iterations
+  resume_from_checkpoint: null # The path to a checkpoint file to continue the training, restores the whole state including the epoch, step, LR schedulers, apex, etc.
 
 model:
-  data_dir: ??? # /path/to/data
-  class_balancing: null # or weighted_loss
+  dataset:
+    do_lower_case: false # true for uncased models, false for cased models, will be set automatically if pre-trained tokenizer model is used
+    max_seq_length: 36
+    class_balancing: null # or weighted_loss
+    use_cache: false
+    num_workers: 2 # number of workers for data loaders
+    drop_last: false # drops the last last batch if it is smaller than the batch size
+    pin_memory: false # enables pin_memory feature of the data loaders
 
   train_ds:
-    prefix: train
+    file_name: ???
     batch_size: 32
     shuffle: true
     num_samples: -1
-    num_workers: 2
-    use_cache: false
-    drop_last: false
-    pin_memory: false
 
   validation_ds:
-    prefix: dev
+    file_name: null
     batch_size: 32
     shuffle: false
     num_samples: -1
-    num_workers: 2
-    use_cache: false
-    drop_last: false
-    pin_memory: false
 
   language_model:
     pretrained_model_name: roberta-base
-    max_seq_length: 36
-    bert_checkpoint: null
-    bert_config: null
     tokenizer: nemobert # 'nemobert' or 'sentencepiece'
+    checkpoint_file: null # the path of a checkpoint file of a BERT-like model if a customized checkpoint is supposed to be used
```

Comment:
can we keep bert_checkpoint main? this name can be confused with the ckpt for the whole model

Reply:
done